.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      
+    - name: Set up Rust
+      uses: actions/setup-rust@v1
```

Comment:
Is this necessary? Doe github's built-in rust toolchain not work?

Reply:
Not sure, I think we need to merge this PR so we can get the CI setup to check if it works